Update WordPress.md (#83)
Laura-Novich-OBW/Tool-Chest: Publishing-tools/WordPress.md

Open source.

### Description:

> A web content management system, one of the most popular in the world. it was used by 42.8% of the top 10 million websites as of October 2021. To function, WordPress has to be installed on a web server, either as part of an Internet hosting service or on a computer running the WordPress software package. It uses different themes and plugins.

Reviewer: ```suggestion
A web content management system, one of the most popular in the world, WordPress is used to publish 42.8% of the top 10 million websites as of October 2021. To function, WordPress has to be installed on a web server, either as part of an Internet hosting service or on a computer running the WordPress software package. It uses different themes and plugins.
```
